sensors/2.0/iiohal_mediation_v2.0/libiio_client/context.c

```diff
@@ -343,13 +360,42 @@ struct iio_context * iio_create_local_context(void)
 struct iio_context * iio_create_network_context(const char *hostname)
 {
 #ifdef WITH_NETWORK_BACKEND
-    return network_create_context(hostname);
+	return network_create_context(hostname);
```

Comment:
Rest of the file uses 4 space indentation whereas the modified codes uses tab for indentation. Can we follow 1 indentation format in the file?